docker-compose.yml

```diff
@@ -9,8 +9,12 @@ services:
     volumes:
       - .:/usr/windbreaker-hooks
       - /usr/windbreaker-hooks/node_modules
+      # uncomment to link and/or replace first path if service utils are located elsewhere
+      # - ../windbreaker-service-util:/usr/windbreaker-hooks/node_modules/windbreaker-service-util
```

Comment:
Added these lines as a linking reference/convenience. I can remove these if people would rather not see them.